app/services/users_service.rb

```diff
@@ -17,6 +17,8 @@ def login(email, password)
   end
 
   def register(email, password, organization_name)
+    return result.not_allowed_failure!(code: 'signup is disabled') if ENV.fetch('LAGO_SIGNUP_DISABLED', 'false') == 'true'
```

Comment:
Style/IfUnlessModifier: Modifier form of `if` makes the line too long. (https://rubystyle.guide#if-as-a-modifier)